web/pages/api/auth/signin/index.js

```diff
@@ -1,6 +1,7 @@
 import nextConnect from 'next-connect'
 import bcrypt from 'bcrypt'
 import { query } from '../../../../utils/query'
+import passport from 'passport'
```

Comment:
We should never end up having to import passport outside of server.js. Once included initialized and setup as middleware, everything we will need will be in the req objects.